### ISSUE TITLE
How to use RPG MV Map

### ISSUE THREAD

**Issue by Majirefy** [pixijs/pixi-tilemap#17]
How to use this for RPG MV Map?

RPG MV creates a JSON file including events, bgm, and other data. Is it possible to use RM like a map editor?

**Comment by ivanpopelyshev** [comment]
Yes, it is possible. Just clone it and look into "index.html" and everything inside "demo" folder.
